ui/packages/consul-ui/app/components/code-editor/layout.scss

```diff
@@ -32,3 +33,121 @@
 %code-editor > pre {
   display: none;
 }
+
+%code-editor .ember-basic-dropdown-trigger {
+  background-color: var(--tone-gray-900);
+  color: var(--black);
+  border-radius: 2px;
+  border: 1px solid;
+  border-color: var(--tone-gray-700);
+  margin: 0 8px;
+  width: 120px;
+  height: 32px;
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+}
```

Comment:
There are a few missing `rgb` here. I dunno if you noticed but in the ColorSwatch docs things I linked to, if you click on the swatch it copies the value (inc. rgb) to the clipboard)

Reply:
Yes, I noticed but I was getting the wrong colours because of the dark mode I guess so I ended up writing manually (the wrong thing). 

Also I noticed that there were some inconsistencies in that file already, see https://github.com/hashicorp/consul/blob/25961355c1492780727a7f45bb41580a6e37ae2a/ui/packages/consul-ui/app/components/code-editor/layout.scss#L18 and https://github.com/hashicorp/consul/blob/25961355c1492780727a7f45bb41580a6e37ae2a/ui/packages/consul-ui/app/components/code-editor/layout.scss#L28

Reply:
Yup only `--tone`s should use the rgb syntax. Any colours without `--tone` should pretty much be ignored.